Gemfile

```diff
@@ -53,7 +53,8 @@ group :admin do
   gem 'axe-core-capybara', '~> 4.7', require: false
 end
 
-group :utils do
+group :lint do
+  gem 'erb-formatter', require: false
   gem 'pry'
   gem 'launchy', require: false
```

Comment:
`pry` and `launchy` aren't strictly linting tools 🤔

Reply:
Agree and I do not see an advantage in having too many bundle groups.

Reply:
I moved both of them up in the main gem group and merge into it also the `:ci` and `:docs` group, I'll keep `:release` and `:lint` because that allows skipping the full bundle install and get shorter CI runs and feedback cycles for PRs.